MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,643 @@
+                e.DataFormItem.DefaultLayoutSettings.LabelWidth = 100;
+                e.DataFormItem.DefaultLayoutSettings.EditorWidth = 200;
+            }
+                
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Background colour
+
+You can change the background colour for the label and editor by using `GenereateDataFormItem` event.
+
+You can change the background colour for the label and editor by using the [Background] `Link` property in [DataFormItem] `Link` .
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.Background = Colors.Azure; 
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+### Setting order for editor
+
+By default the editor can be placed in the order of how you declare a propety in MainPage.xaml.cs
+
+You can change the order of item in a row only if the column count more than 1 [If the column count = n , you can place a item upto n-1 position].
+
+You can change the order of item by using [ItemsOrderInRow] `Link` property in [DataFormItem] `Link`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.ColumnCount = 6;
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.ItemsOrderInRow = 3;  //Place a item upto 5 
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Label icon
+
+You can set label icon instead of using label text. Example:- You have a PhoneNumber editor, and the editor label is PhoneNumber. Here You can use a phone image for the label.
+
+`Add a phone image in Images folder`
+
+`build action as Embedded res`
+Right-click on the phone image and change the build action to "Embedded resource" and configure an image in.csproj. Directly add the name of the image by using the [LeadingLabelIcon] `Link` property in [DataFormItem] `Link`.
+
+Supported image format:- .png and .svg
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "PhoneNumer")
+            {
+                e.DataFormItem.LeadingLabelIcon = "phone.png"
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+Without adding an image to the Image folder, you can give the full path of the image.
```

Comment:
No need, just mention the porperty type is ImageSource and code with one example

Reply:
codes removed